rust-src/concordium_base/benches/serialization_benches.rs

```diff
@@ -0,0 +1,58 @@
+#[macro_use]
```

Comment:
Can you perhaps throw a few words on the benchmark going on there. What is the motivation and what is it used for.